### PR TITLE
Add clarity, fix English grammar in API doc

### PR DESCRIPTION
**Pull request by jedmao**

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -126,7 +127,7 @@ cleaner.process(css, options);
 postcss([ cleaner(options) ]).process(css);
 ```
 
-Asynchronous plugin should return `Promise` instance.
+Asynchronous plugin should return a `Promise` instance.
```
Comment (ben-eb) [inline docs/api.md]: > Asynchronous plugin**s**

File: docs/api.md [hunk 2]
```diff
@@ -156,7 +157,7 @@ postcss.plugin('postcss-caniuse-test', function () {
 });
 ```
 
-Send data to next plugins using the [`Result#messages`] array.
+Send data to the next plugins using the [`Result#messages`] array.
```
Comment (ben-eb) [inline docs/api.md]: Rather than "the next" can we put "other" here?

Comment (jedmao) [inline docs/api.md]: Yep!

File: docs/api.md (hunk 1 above)
Comment (jedmao) [inline docs/api.md]: Done.

**Comment by ben-eb** [comment]
Thanks. :+1:

**Pull request merged**